DuckDuckGo/MainViewController.swift

```diff
@@ -1774,4 +1788,49 @@ extension MainViewController: VoiceSearchViewControllerDelegate {
     }
 }
 
+
+// MARK: - History UIMenu Methods
+
+extension MainViewController {
+
+    private func refreshBackForwardMenuItems() {
+        guard let currentTab = currentTab, #available(iOS 14.0, *) else {
+            return
+        }
+        omniBar.backButton.menu = historyMenu(with: currentTab.webView.backForwardList.backList.reversed())
+        backButton.menu = historyMenu(with: currentTab.webView.backForwardList.backList.reversed())
+        
+        omniBar.forwardButton.menu = historyMenu(with: currentTab.webView.backForwardList.forwardList)
+        forwardButton.menu = historyMenu(with: currentTab.webView.backForwardList.forwardList)
```

Comment:
Are we able to build each of these menus once and share them? Something like:

```suggestion
        let backMenu = historyMenu(with: currentTab.webView.backForwardList.backList.reversed())
        omniBar.backButton.menu = backMenu
        backButton.menu = backMenu
        
        let forwardMenu = historyMenu(with: currentTab.webView.backForwardList.forwardList)
        omniBar.forwardButton.menu = forwardMenu
        forwardButton.menu = forwardMenu
```

Reply:
Oh, much better. This happened because of a "I forgot about the iPad" moment, and didn't look it back to make it better. Thank you :)